code/modules/clothing/masks/gasmask.dm

```diff
@@ -37,7 +35,9 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 
 /obj/item/clothing/mask/gas/Initialize(mapload)
 	. = ..()
-	init_fov()
+
+	if(flags_cover & PEPPERPROOF)
```

Comment:
so a SWAT mask is no longer good for riot control?

Reply:
syndicate, ninja mask, `/obj/item/clothing/mask/gas/sechailer/plasmaman` too maybe?